### PR TITLE
Jb57615

### PR DESCRIPTION
**Pull request by martyone**

### CODE REVIEW

**Comment by martyone** [comment]
The very first two commits belong to other PR.

File: sdk-setup/src/mb2 [hunk 1]
```diff
@@ -51,7 +51,32 @@ usage() {
...
+   build-requires  Update build time dependencies
+   prepare  Prepare sources using the recipe from spec file
+   apply  Apply patches listed in the spec file
+   build  Execute all build steps at once
+   qmake  Execute qmake build step
+   cmake  Execute cmake build step
+   make  Execute make build step
+   make-install  Execute make-install build step
+   package  Execute package creation build step
+   build-shell  Execute custom build steps
```
Comment (vigejolla) [inline sdk-setup/src/mb2:69]: I think the term "custom build step" is a bit misleading here. The other build steps are defined in RPM spec files, while build-shell command can be used for executing arbitrary commands. They don't have to be build steps. You don't even need a spec file.

File: sdk-setup/src/mb2 [hunk 2]
```diff
@@ -51,11 +51,275 @@ usage() {
     less --quit-if-one-screen <<EOF
 $(synopsis)
 
-Executes all or a subset of build commands in the context of an rpmbuild.
+mb2 provides convenient interface to Scratchbox 2, rpmbuild and more.
 
-<specfile> will be looked for in the current rpm/ dir. If there is more than
-one it must be provided.  $ME is aware of spectacle and will update the spec
-file if there is an obvious yaml file which is newer.
+COMMANDS OVERWIEW
```
Comment (vigejolla) [inline sdk-setup/src/mb2:56]:
```suggestion
COMMANDS OVERVIEW
```

**Review by vigejolla — CHANGES_REQUESTED** [review]
One typo fix, also please consider if you could improve the `build-shell` overview.

File: sdk-setup/src/mb2 (hunk 1 above)
Comment (martyone) [reply sdk-setup/src/mb2:69]: I don't see a better option. See how it aligns with Qt Creator > Projects > Build > Build Steps > Add Build Step > {qmake, Make, ..., Custom Process Step}, and see how it aligns internally - the user is allowed to execute standard build steps which operate according to .spec, and they are also allowed to executed custom build steps. Whatever is done with the help inside the build shell is considered a build step as building software is the use case we are after here, as the name also suggests.

File: sdk-setup/src/mb2 (hunk 2 above)
Comment (martyone) [reply sdk-setup/src/mb2]: Fixed, thanks

**Review by vigejolla — APPROVED** [review]

**Pull request merged**